src/config.js

```diff
@@ -0,0 +1,36 @@
+(function (angular) {
+    'use strict';
+
+    var module = angular.module('angular-token-auth.route-config', [
```

Comment:
This is named `route-config` but it does more than just the routes, and the filename is `config.js` yet it does routes. Can this be split into `.routes` and `.config` separate file modules?